player-backends/player-backend-gstreamer-vaapi/gt-player-backend-gstreamer-vaapi.c

```diff
@@ -0,0 +1,295 @@
+/*
+ *  This file is part of GNOME Twitch - 'Enjoy Twitch on your GNU/Linux desktop'
+ *  Copyright © 2017 Vincent Szolnoky <vinszent@vinszent.com>
```

Comment:
You should add your own name in the copyright notice, since (presumably) you have written this file. Same for the other files you have written that contain a copyright notice.